bucket4j-spring-boot-starter/src/main/java/com/giffing/bucket4j/spring/boot/starter/config/cache/redis/jedis/JedisCacheListener.java

```diff
@@ -52,42 +46,13 @@ public JedisCacheListener(JedisPool jedisPool, String cacheName, Class<K> keyTyp
 	}
 
 	public void subscribe() {
-		Thread thread = new Thread(() -> {
```

Comment:
Was it intended to revert the changes from [https://github.com/Edwin9292/bucket4j-spring-boot-starter/pull/5](https://deref-web.de/mail/client/GzqxV05Trds/dereferrer/?redirectUrl=https%3A%2F%2Fgithub.com%2FEdwin9292%2Fbucket4j-spring-boot-starter%2Fpull%2F5) ?

Reply:
No that was not intended at all. That's weird, looks like something went wrong with merging. Glad you caught it, will fix it right now

Reply:
Looked into it, appears a revert before the merging had unexpected behavior that i missed. I reverted the revert, so it's fixed now. Marking this as resolved.